packages/native-stack/src/types.tsx

```diff
@@ -351,6 +375,21 @@ export type NativeStackNavigationOptions = {
    * @platform ios
    */
   statusBarStyle?: ScreenProps['statusBarStyle'];
+  /**
+   * Sets the translucency of the status bar. Defaults to `false`.
+   *
+   * @platform android
+   */
+  statusBarTranslucent?: boolean;
+  /**
+   * Sets the direction in which you should swipe to dismiss the screen.
+   * When using `vertical` option, options `fullScreenSwipeEnabled: true`, `customAnimationOnSwipe: true` and `stackAnimation: 'slide_from_bottom'` are set by default.
+   * The following values are supported:
+   * - `vertical` – dismiss screen vertically
+   * - `horizontal` – dismiss screen horizontally (default)
+   * @platform ios
+   */
+  swipeDirection?: ScreenProps['swipeDirection'];
```

Comment:
Could we rename it here to `gestureDirection` to align with [JS stack](https://reactnavigation.org/docs/stack-navigator/#gesturedirection)?

Reply:
But this will need changes upstream as well, no? Or you want to alias this?

Reply:
Nope, you can do it the same as with e.g. [`orientation`](https://github.com/react-navigation/react-navigation/blob/0b01cb0412d8c2cbc6538b0451dd1856474b7276/packages/native-stack/src/views/NativeStackView.native.tsx#L206)

Reply:
Ok, thats what I thought.

Reply:
Done